cs/images.texy

```diff
@@ -163,7 +163,7 @@ $image->send(Image::PNG);
 \--
 
 .[note]
-Protože vykreslení obrázku tím, že dáte odkaz na prezenter, který zavolá $image->send(); nemusí být zrovna nejrychlejší, je lepší si změněný obrázek někam uložit a pak zobrazovat přímo ten uložený obrázek, bez použití $image->send(); Pro inspiraci se můžete podívat "http://forum.nette.org/cs/20563-generovani-vice-variant-obrazku-best-practice":na fórum.
+Protože vykreslení obrázku tím, že dáte odkaz na prezenter, který zavolá $image->send(); nemusí být zrovna nejrychlejší, je lepší si změněný obrázek někam uložit a pak zobrazovat přímo ten uložený obrázek, bez použití $image->send(); Pro inspiraci se můžete podívat na fórum http://forum.nette.org/cs/20563-generovani-vice-variant-obrazku-best-practice.
 
```

Comment:
I tried that too but i liked it better if i can see the actual link in the text. Whatever i've changed that.